### PR TITLE
Support for Laravel Blade file extension

### LINKED ISSUE #83: blade.php is not supported
In Laravel projects `blade.php` files is not supported. i.e there is no autocomplete list of CSS classes.
Version: 1.9.0, VScode 1.15.1, Ubuntu Linux 16.04, Laravel 5.4 Project.

### PR DESCRIPTION
**Pull request by m2de**
Resolves #83

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -125,31 +125,27 @@ export async function activate(context: vscode.ExtensionContext): Promise<void>
...
+        'javascript',
+        'javascriptreact',
+        'erb',
+        'handlebars',
+        'ejs'
+    ];
+    
+    extensions.forEach((extension) => {
+        context.subscriptions.push(
+            provideCompletionItemsGenerator(extension, htmlRegex)
```
Comment (zignd) [inline src/extension.ts]: The same regex can't be applied to all the extensions.

**Comment by zignd** [comment]
Thanks for the contribution, but can you fix that mistake?

File: src/extension.ts (hunk 1 above)
Comment (m2de) [reply src/extension.ts]: My bad, I didn’t spot that and will amend. Thanks.

**Comment by m2de** [comment]
Hi @zignd. Changes have now been implemented and jsx and html regex are applied to correct extensions. Please let me know if I missed anything. Thanks.

**Comment by zignd** [comment]
Good to go now! I will release a new version with the change in a moment.

**Comment by henriquebs12** [comment]
Still got the same problem. Was it solved? What do I have to do?

**Pull request merged**